beekeeper-cleanup/src/test/java/com/expediagroup/beekeeper/cleanup/service/PagingCleanupServiceTest.java

```diff
@@ -1,225 +1,83 @@
-/**
```

Comment:
This shouldn't be removed. I'd also suggest running the full Maven lifecycle over all the code as this should add the license headers to all the new files for you.

Reply:
not sure how that got removed - either way added all license headers